pyproject.toml

```diff
@@ -94,3 +94,6 @@ exclude_lines = [
   "if TYPE_CHECKING:",
   "return NotImplemented",
 ]
+
+[tool.bandit]
+skips = ["B404", "B603"]
```

Comment:
Since this is an upstream enforced disablement, we should put each on a separate line and add a commentary with a short explanation AND/OR a URL to the official docs on the rule

Reply:
I thought the git blame command would be sufficient :smile_cat:

Reply:
I mean, yes, RTFM is a universally applicable rule and yet programmers write all sorts of commentaries all around the source code to spare other poor soul's time ;) . In general, one also can't universally rely on commit messages providing any further information - "accidents happen" and commit messages end up empty (not our case), it's just a good practice IMO that costs us nothing.